### PR TITLE
New Package: ASRock.RestartToUEFI version 1.0.12

### PR DESCRIPTION
**Pull request by loicreynier**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137632)

### CODE REVIEW

**Comment by loicreynier** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Hi @loicreynier,

This package installed normally on a Windows 10 VM. But when launched, it did not seem to run - after the UAC prompt, there was no window, console output, or error in either the error log nor its own folder. 

Is this expected?

**Comment by loicreynier** [comment]
@stephengillie thanks for the feedback.

This is not expected. A window should spawn after the UAC prompt. It worked fine on my Windows 11 system with `winget install --manifest <path>`.
The program being related to UEFI, it might not work on a VM.

File: manifests/a/ASRock/RestartToUEFI/1.0.12/ASRock.RestartToUEFI.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: ASRock.RestartToUEFI
+PackageVersion: 1.0.12
+PackageLocale: en-US
+Publisher: ASRock
```
Comment (Trenly) [inline manifests/a/ASRock/RestartToUEFI/1.0.12/ASRock.RestartToUEFI.locale.en-US.yaml:7]:
```suggestion
Publisher: ASRock Inc.
```
This should match Control Panel

File: manifests/a/ASRock/RestartToUEFI/1.0.12/ASRock.RestartToUEFI.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: ASRock.RestartToUEFI
+PackageVersion: 1.0.12
+PackageLocale: en-US
+Publisher: ASRock
+PublisherUrl: https://www.asrock.com
+PackageName: Restart to UEFI
```
Comment (Trenly) [inline manifests/a/ASRock/RestartToUEFI/1.0.12/ASRock.RestartToUEFI.locale.en-US.yaml:9]:
```suggestion
PackageName: ASRock Restart to UEFI
```
This should match as close as possible to Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
> @stephengillie thanks for the feedback.
> 
> This is not expected. A window should spawn after the UAC prompt. It worked fine on my Windows 11 system with `winget install --manifest <path>`. The program being related to UEFI, it might not work on a VM.

It's possible that this package might require Windows 11. So far, I've been validating on Windows 10.

**Comment by loicreynier** [comment]
> It's possible that this package might require Windows 11. So far, I've been validating on Windows 10.

Indeed on ASRock website this specific version (latest) is listed as a Windows 11 software while other versions are listed as Windows 10 software.

File: manifests/a/ASRock/RestartToUEFI/1.0.12/ASRock.RestartToUEFI.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ASRock.RestartToUEFI
+PackageVersion: 1.0.12
+InstallerType: zip
```
Comment (stephengillie) [inline manifests/a/ASRock/RestartToUEFI/1.0.12/ASRock.RestartToUEFI.installer.yaml:6]:
```suggestion
InstallerType: zip
MinimumOSVersion: 10.0.22621.0
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> ERROR: Signature Update failed with hr=0x80070652 CmdTool: Failed with hr = 0x80070652. Check C:\Users\VALIDA~1\AppData\Local\Temp\MpCmdRun.log for more information  Signature Update Failed. Failure to parse win 10, attempting win 11 Installation failed with exit code -1978335216

(Automated response - build 727.)

**Comment by stephengillie** [comment]
This package installed normally on a Windows 11 VM. But installing it caused the VM to pause due to a critical I/O error. The VM wasn't recoverable and had to be reverted to snapshot to become operational again. Tried twice with the same results. 

image

**Comment by stephengillie** [comment]
After clearing the host issue, Windows 11 presented identical behavior to Windows 10: 

> This package installed normally on a Windows 10 VM. But when launched, it did not seem to run - after the UAC prompt, there was no window, console output, or error in either the error log nor its own folder.

**Pull request closed without merging**